apps/omg_child_chain/lib/omg_child_chain/release_tasks/set_fee_buffer_duration.ex

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ChildChain.ReleaseTasks.SetFeeBufferDuration do
+  @moduledoc false
+  use Distillery.Releases.Config.Provider
+  require Logger
+
+  @app :omg_child_chain
+  @config_key :fee_buffer_duration_ms
+  @env_name "FEE_BUFFER_DURATION_MS"
```

Comment:
I find it misleading to the environment but ENV variable is meant
```suggestion
  @env_var_name "FEE_BUFFER_DURATION_MS"
```